_drafts/json-api.markdown

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: JSON API
+excerpt: JSON API is a specification for building APIs on the web.
+description: JSON API is a specification for building APIs on the web.
+---
+
+[JSON API][json-api] is a specification for building APIs on the web.
+When designing an API that communicates via JSON, developers often get tangled in the minutia of how requests and responses with be formed.
+Significant time may be spend belaboring precisely how the JSON will be formed and even then minor discrepancies may crop up between contributions from different people.
+JSON API addresses these issues by establishing a specification that everyone can agree on.
+The JSON API team likes to call it an "anti-bikeshedding weapon".
```

Comment:
Perhaps link here to what "bikeshedding" means, I only recently learned it myself.